### PR TITLE
Update environment-variables.md

### PR DESCRIPTION
**Pull request by d-stefanovic**

### CODE REVIEW

**Review by csidyel — APPROVED** [review]

**Comment by DamjanBecirovic** [comment]
I would suggest rewording this a bit to make it easier for reading:

> Semaphore supports setting environment variables for an [individual job](envvars-perjob), for [all jobs in a block](envvars-perblock), and for [all jobs in the pipeline](global_job_config]).

The markdown for this is below:

```
Semaphore supports setting environment variables for an [individual job][envvars-perjob], for [all jobs in a block][envvars-perblock], and for [all jobs in the pipeline][global_job_config].
```

**Comment by d-stefanovic** [comment]
@DamjanBecirovic thanks for the suggestion, it's been changed.

**Review by DamjanBecirovic — APPROVED** [review]
